### DISCUSSION TITLE
Autocomplete classes in PHPStorm

### DISCUSSION THREAD

**Discussion by bixgomez** [gregsullivan/_tw#43] [General]
I am a PHPStorm user, currently on 2021.3.3, which might be the problem...

I have the bundled TailwindCSS plugin activated.  If I create a new HTML file, I get Tailwind class hints, but not in any PHP file.

The class hints that I do get in HTML seem limited -- leading me to believe this is a version issue!

**Comment by gregsullivan** [comment]
I use 2022.3.2, and I don't have this issue. That said, I only returned to PhpStorm at the start of 2023, so I can't report on whether or how well it worked with past versions.

**Reply by bixgomez** [reply]
Yeah, must be a version issue.  I've been a loyal PHPStorm user until recently, when I had to revert to an earlier version due to a license I inherited from someone after mine expired.  I've been trying to warm to VSCode, but it's been difficult, so I tend to move back and forth.  I did find a VSC theme that matches PHPStorm's colors, so that's helpful!  And it's nice to not have to pay an annual fee...

**Reply by gregsullivan** [reply]
I had very heavily modified my VS Code preferences, but everything fell apart due to this:

https://github.com/iocave/customize-ui/issues/156

I'd been locked to an old version of VS Code, but I didn't think that was sustainable, so I switched to PhpStorm. I really, really like it now. I'd tried it in the past, but it hadn't stuck for me. The new UI combined with core plugins covering all of my needs makes a huge difference for me.
